.npmrc

```diff
@@ -0,0 +1 @@
+//registry.npmjs.org/:_authToken=${NPM_TOKEN}
```

Comment:
I hope it doesn't affect dev environment

Reply:
it affects :C but we need it for publish